common/changes/@uifabric/merge-styles/style-to-classname_2017-10-31-23-57.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/merge-styles",
+      "comment": "Fix mergeStyles generates wrong CSS rules for registered classnames as selector values",
```

Comment:
I don't know what scenario this is fixing. The PR comments don't mention what's going fixed either.

Reply:
Updated description